src/lib/forms-schema/elements/ArcGISWebMapElement.ts

```diff
@@ -67,4 +67,25 @@ export default Joi.object({
       longitude: Joi.number().required(),
     }),
   }),
+  snapshotImagesEnabled: Joi.boolean().default(false),
+  minSnapshotImages: Joi.alternatives([
+    Joi.number().min(0),
+    Joi.object({
+      type: Joi.string().valid('FORM_ELEMENT').required(),
+      elementId: Joi.string().uuid().required(),
+    }),
+  ]),
```

Comment:
The types currently only support number. If we want to support dynmaic min/max, we need to update the types as well. Prob not worth it 🤷

Reply:
ah shit. just lazily copied from repeatable sets